README.md

```diff
@@ -44,9 +44,11 @@ This file contains a number of front-end interview questions that can be used wh
 * If you could master one technology this year, what would it be?
 * What are the differences between Long-Polling, Websockets and SSE?
 * Explain the importance of standards and standards bodies.
-* What is FOUC? How do you avoid FOUC?
+* What is Flash of Unstyled Content (FOUC)? How do you avoid FOUC?
 * Do your best to describe the process from the time you type in a website's URL to it finishing loading on your screen.
-* Explain what ARIA and screenreaders are, and how to make a website accessible.
+* What is your approach to accessiblity and developing in keyboard accessibility and visual focus, such as using foundational HTML and Javascript techniques? 
+* Explain your experience with Web Accessibility Initiative's Accessible Rich Internet Applications (WAI-ARIA): Do you know the difference between developing an aceesible website following Web Content Accessibility Guidelines (WCAG) web standards and custom developing for assistive technologies such as screenreaders with WAI-ARIA?
+* What experience do you have in developing in frameworks such as Microsoft Active Accessibility (MSAA), Microsoft UI Automation (UIA), or Java Accessibilty Bridge, or Linux Foundation's IAccessible2?
```

Comment:
Absolutely. If they do not have knowledge of these concepts, they have no
experience in addressing accessibility or usability for people with
disabilities. That is to say they probably rely upon "whatever is in the
framework" and don't understand or are honestly ignorant on the topic.

On Tue, Jan 20, 2015 at 3:37 PM, Rich Gilbank <notifications@github.com>
wrote:

> In README.md
> <https://github.com/h5bp/Front-end-Developer-Interview-Questions/pull/226#discussion-diff-23253470>
> :
>
> >  * Do your best to describe the process from the time you type in a website's URL to it finishing loading on your screen.
> > -* Explain what ARIA and screenreaders are, and how to make a website accessible.
> > +* What is your approach to accessiblity and developing in keyboard accessibility and visual focus, such as using foundational HTML and Javascript techniques?
> > +* Explain your experience with Web Accessibility Initiative's Accessible Rich Internet Applications (WAI-ARIA): Do you know the difference between developing an aceesible website following Web Content Accessibility Guidelines (WCAG) web standards and custom developing for assistive technologies such as screenreaders with WAI-ARIA?
> > +* What experience do you have in developing in frameworks such as Microsoft Active Accessibility (MSAA), Microsoft UI Automation (UIA), or Java Accessibilty Bridge, or Linux Foundation's IAccessible2?
>
> Are these relevant to front-end development?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/h5bp/Front-end-Developer-Interview-Questions/pull/226/files#r23253470>
> .
>



-- 
(703) 942 9569 google voice 24x7
"You have access to the greatest source of knowledge in the universe."
"Well, I do talk to myself sometimes." ~ The Invasion of Time (1978)